guardian/subscriptions-frontend
# TCFv2 - privacy settings link in footer

## What does this change?
Adds a link to the footer which surfaces the privacy settings modal, allowing the user to adjust their privacy preferences at any time.

## How to test
Run the site as normal. With JS enabled, a 'Privacy Settings' link should be visible in the footer, and should open the privacy settings manager modal

## Images

**Mobile**
![Screenshot 2020-09-07 at 15 14 56](https://user-images.githubusercontent.com/29146931/92396576-237d9600-f11d-11ea-81e3-e94023046edb.png)

**Tablet**
![Screenshot 2020-09-07 at 15 15 41](https://user-images.githubusercontent.com/29146931/92396583-24aec300-f11d-11ea-962c-e99055f89a22.png)

**Desktop**
![Screenshot 2020-09-07 at 15 12 52](https://user-images.githubusercontent.com/29146931/92396587-27a9b380-f11d-11ea-8d0e-9cf2dd6872c3.png)
